docs/en/infraops/infrastructure-metrics.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[infrastructure-metrics]]
+[role="xpack"]
+
+== Infrastructure metrics
+
+The metrics listed below are provided by the {beats} shippers.
+// ++ what does the sentence below mean? What does the reader need to do?
+Each system type requires their corresponding identity field to be in the same event document:
+
+* Hosts require `host.name`
+* Docker containers require `container.id`
+* Kubernetes pods require `kubernetes.pod.uid`
+
+// + I think this bit applies to the View Metrics page, not the Infrastructure home page.
+// ++ What does this mean, and what does the reader need to do about it, if anything to ensure the "required field" is available?
+For the metrics detail page, `event.dataset` is a required field. This field is a combination of `metricset.module`, which is the Metricbeat module name, and `metricset.name`, which is the metricset name.
+
+[float]
+=== Host Metrics
+
+*CPU Usage*:: Average of `system.cpu.user.pct` added to the average of `system.cpu.system.pct` divided by `system.cpu.cores`
+
+*Memory Usage*:: Average of `system.memory.actual.used.pct`
+
+*Load*:: Average of `system.load.5`
+
+*Inbound Traffic*:: Derivative of the maximum of `system.network.in.bytes` scaled to a 1 second rate
+
+*Outbound Traffic*:: Derivative of the maxoimum of `system.network.out.bytes` scaled to a 1 second rate
+
+*Log Rate*:: Derivative of the cumulative sum of the document count scaled to a 1 second rate.
+This metric relies on the same indices as the logs.
+
+[float]
+=== Docker Container Metrics
+
+*CPU Usage*:: Average of `docker.cpu.total.pct`
+
+*Memory Usage*:: Average of `docker.memory.usage.pct`
+
+*Inbound Traffic*:: Derivative of the max of `docker.network.in.bytes` scaled to a 1 second rate
```

Comment:
Not the biggest deal, but you start out using "maximum", then switch to "max" from here on. Any reason why?

Reply:
Good spot!